cs/v1/CS_v1.0.0.yaml

```diff
@@ -396,6 +416,17 @@ paths:
                 example: 1.0.0
               description: |
                 SemVer used to indicate the version of the contract (API version) returned.
+            Link:
+              schema:
+                type: string
+                example : <https://api.dcsa.org/v1/vessel-schedules?limit=20&cursor=f8dcb4c4-3fba-41ef-9991-2be97dad57dc>; rel="next",<https://api.dcsa.org/v1/vessel-schedules?limit=20&cursor=f8dcb4c4-3fba-41ef-9991-2be97dad57dc>; rel="prev"
+              description: |
+                The `Link` header contains links to related pages. Each link has a relation type (`rel`) that indicates the relation of the linked page to the current page. The following are the allowed `Link` header relation types:
+
+                  - `first` - An optional link to the first page.
+                  - `prev`  - An optional link to the previous page. The prev header link MUST be omitted if the current page is the first page.
+                  - `next`  - A link to the next page. The next header linka MUST be omitted if the current page is the last page, otherwise it MUST be provided.
```

Comment:
```suggestion
                  - `next`  - A link to the next page. The next header link MUST be omitted if the current page is the last page, otherwise it MUST be provided.
```